src/pages/ClusterPage/ClusterPage.js

```diff
@@ -250,37 +256,18 @@ export default class ClusterPage extends Component {
 
 					{cluster.status === 'active' || cluster.status === 'deployments in progress' ? (
 						<div>
-							{cluster.trial || cluster.subscription_id ? (
+							{isUsingClusterTrial || cluster.subscription_id ? (
 								<Link to={`/clusters/${cluster.id}`}>
 									<Button type="primary">View Details</Button>
 								</Link>
-							) : null}
-							{cluster.trial || cluster.subscription_id ? (
```

Comment:
Did you confirm this change ? I guess this was used to fire Payment Modal based on URL.

Reply:
We should remove them. We needed this deployed and there was debugging needed, but I missed removing them prior to commiting.